libs/wingcompiler/src/compile.ts

```diff
@@ -63,6 +63,9 @@ export interface CompileOptions {
 
   // target directory for the output files
   readonly targetDir?: string;
+
+  // overrides the target directory for the output files
+  readonly output?: string;
```

Comment:
```suggestion
  readonly outputOverride?: string;
```

Add description of the default behavior to docstrings

Reply:
I think there is a value in keeping it aligned to the cli (we pass all the options with a spread operator) 
looked at the cli tool and there is no way to change the variable :(